Makefile

```
@@ -21,6 +25,11 @@ container: build ## Build the container
 	@docker -v || (code=$$?; printf "\033[0;31mError\033[0m: there was a problem with Docker\n"; exit $$code)
 	docker build --build-arg VERSION=$(VERSION) --build-arg GIT_COMMIT=$(GIT_COMMIT) --build-arg DATE=$(DATE) --target $(TARGET) -f build/Dockerfile -t $(PREFIX):$(TAG) .
 
+.PHONY: container
+agent-container: # Build the agent container
```

this prevents `make` from printing help for agent-container
```suggestion
agent-container: ## Build the agent container
```